tests/codeception/acceptance/users_frontend.feature

```diff
@@ -50,3 +50,8 @@ Feature: Users Frontend
     And User "User Three" did login at least once
     When I login as a super admin from backend
     Then I should see last login date for "User Three"
+
+Scenario: 
+	Delete user Given I have a user with user name "User Three" 
+	When I Delete the user "User Three" 
+	Then I should see "No Matching Results" for deleted user "User Three"
```

Comment:
Small typo: Should be

```Gherkin
Scenario: Delete user
    Given I have a user with user name "User Three" 
    When I Delete the user "User Three" 
    Then I should see "No Matching Results" for deleted user "User Three" 
```

Please remember the newline at the end of the file!